setup.py

```diff
@@ -44,12 +44,12 @@ def read(*rnames):
           'Programming Language :: Python',
           'Operating System :: OS Independent',
           'Programming Language :: Python :: 3',
-          'Programming Language :: Python :: 3.7',
```

Comment:
You'll also want to update the `python_requires=">=3.7, <3.14",` line if you're dropping 3.7.

(I've released check-python-versions that knows about 3.13 just now.)

Reply:
Thanks - I think the problem was that I did not use the upgrade script that's part of zope.meta, I just did it by hand...

Reply:
Also, thanks for the release, it's all green now.